xml2rfc/__init__.py

```diff
@@ -54,8 +54,7 @@ def get_versions():
             try:
                 dist = metadata.distribution(p)
                 versions.append((dist.metadata['name'], dist.metadata['version']))
-            except Exception as e:
-                print(e)
+            except:
```

Comment:
This will silently bury potentially more than "not installed". Is there a specific exception you could look for to silence the print rather than any exception?

Reply:
This is used to get the versions of the dependencies. I think it's best to silently burry everything.